CHANGELOG.md

```diff
@@ -75,6 +75,7 @@
 * Fixed an issue where lane guidance icons would sometimes highlight the wrong arrow.([#2942](https://github.com/mapbox/mapbox-navigation-ios/pull/2942))
 * The duration annotations added by the `NavigationMapView.showRouteDurations(along:)` method are now set in the fonts you specify using the `NavigationMapView.routeDurationAnnotationFontNames` property. Use this property to specify a list of fallback fonts for better language support. ([#2873](https://github.com/mapbox/mapbox-navigation-ios/pull/2873))
 * Removed deprecated and obsoleted `EventsManager` typealias, `StatusView.delegate`, `StatusView.canChangeValue`, `RouteLegProgress.upComingStep`, `StatusViewDelegate`, `DeprecatedStatusViewDelegate` and `BottomBannerViewController.init(delegate:)`. ([#2993](https://github.com/mapbox/mapbox-navigation-ios/pull/2993))
+* `NavigationEventsManager` now allows the application to include application metadata `appMetadata` such as the application name and version in feedback events by setting `withAppMetadata` parameter of  `NavigationEventsManager.recordFeedback()` to `true`. ([#3007](https://github.com/mapbox/mapbox-navigation-ios/pull/3007)).
```

Comment:
Overall looks good to me. Would it be possible to implement basic unit-test for newly implemented functionality? Please also rebase onto `release-v2.0` to fix outstanding conflicts.

Reply:
Tests are welcomed.

Reply:
We can't test against the schema directly, and unit-tests just verify that it outputs some JSON that _looks_ correct. We have to go through a test app to check whether it is working properly. Currently doing so with @bamx23.

Reply:
We can verify that `userInfo` is propagated to `MMEEventsManager` as intended.